cmd/main.go

```diff
@@ -41,6 +41,7 @@ func newPage() Page {
 func main() {
 
 	e := echo.New()
+	e.Static("/public", "public")
```

Comment:
Serve `/public` directory for public content.

Reply:
I don't do a lot of frontend, but maybe /static would be a more representative name? /public works too, especially if it's commonly used.

Reply:
both are used, doesn't really matter.